src/aspire/volume/symmetry_groups.py

```diff
@@ -182,15 +171,16 @@ def generate_rotations(self):
 
         :return: Rotation object containing the Dn symmetry group and the identity.
         """
+
         # Rotations to induce cyclic symmetry
-        angles = 2 * np.pi * np.arange(self.order, dtype=self.dtype) / self.order
-        rot_z = Rotation.about_axis("z", angles).matrices
+        angles = 2 * np.pi * np.arange(self.order) / self.order
+        rot_z = Rotation.about_axis("z", angles, dtype=np.float64).matrices
```

Comment:
same concern as above. Just check if angles are cast before the `about_axis` conversion or after by this `dtype=`. We always want the compute in doubles if we're doing this...

Reply:
see above